lib/prop/options.rb

```diff
@@ -17,8 +17,27 @@ def self.build(options)
       raise RuntimeError.new("Invalid threshold setting") unless result[:threshold] > 0
       raise RuntimeError.new("Invalid interval setting")  unless result[:interval] > 0
 
+      if result.key?(:increment)
+        result[:increment] = result[:increment].to_i
+        raise RuntimeError.new("Invalid increment setting") unless result[:increment] > 0
```

Comment:
... even better: `raise ArgumentError, ":increment must be a positive Integer" if !result[:increment].is_a?(Fixnum) || !result[:increment] > 0`